src/main/java/org/ecommerce/models/Order.java

```diff
@@ -2,25 +2,30 @@
 
 import lombok.Getter;
 import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+import org.ecommerce.enums.OrderStatus;
 
 import java.sql.Date;
 import java.util.List;
 
+@SuperBuilder
 @Setter
 @Getter
 public class Order extends Identity {
     private Long customerId;
     private Date orderDate;
     private List<Product> products;
+    private OrderStatus status;
     private ShippingInformation shippingInformation;
     private BillingInformation billingInformation;
     private PaymentDetails paymentDetails;
 
-    Order(Long id, Long customerId, Date orderDate, ShippingInformation shippingInformation, BillingInformation billingInformation, PaymentDetails paymentDetails, List<Product> products) {
+    Order(Long id, Long customerId, Date orderDate,  List<Product> products, OrderStatus status, ShippingInformation shippingInformation, BillingInformation billingInformation, PaymentDetails paymentDetails) {
```

Comment:
Try replacing with lombok annotation `@RequiredArgsConstructor `

Reply:
Apply the same logic to other boilerplate constructors in the project

Reply:
Due to the classes architecture I had to use some more lombok annotations besides ```@RequiredArgsConstructor```. I already deleted some constructors, changes can be visualized in the following commit: https://github.com/algutierrez-griddynamics/e-commerce/pull/24/commits/ab5c86a11eb53111e6d406754ff32882978a13ca